var/spack/repos/builtin/packages/environment-modules/package.py

```diff
@@ -59,13 +59,15 @@ class EnvironmentModules(Package):
 
     variant("X", default=True, description="Build with X functionality")
 
-    depends_on("autoconf", type="build", when="@main")
-    depends_on("automake", type="build", when="@main")
-    depends_on("libtool", type="build", when="@main")
-    depends_on("m4", type="build", when="@main")
-    depends_on("python", type="build", when="@main")
-    depends_on("py-sphinx@1.0:", type="build", when="@main")
-    depends_on("gzip", type="build", when="@main")
+    depends_on("less", type="build")
```

Comment:
Is this for the build or runtime?

Reply:
Still need to check, but my guess is a configure script needs it. Period. For what is worth it's not mentioned in:

https://modules.readthedocs.io/en/latest/INSTALL.html